solidity/contracts/helpers/TestBancorConverter.sol

```diff
@@ -12,7 +12,6 @@ contract TestBancorConverter is BancorConverter {
         BancorConverter(_token, _registry, _maxConversionFee, _reserveToken, _reserveWeight)
         public
     {
-        etherToken = IEtherToken(0);
```

Comment:
don't we still need this?

Reply:
@yudilevi : I though about it for a while. Definitely not `0` (if anything, the real address, i.e., 0xeee...). But it's not really used anyway (i.e., every test which uses this helper contract, calls the setter function immediately after instantiating the contract). I'll do another sweep in order to make sure of that.

Reply:
ok cool